### DISCUSSION TITLE
Option to download as .xlsx

### DISCUSSION THREAD

**Discussion by smilingthax** [evidence-dev/evidence#2330] [Ideas]
## Feature Description

Currently, the data Downloads links produce CSV files. It would be nice to be able to download Excel (XLSX) files.

## Goal of Feature

Customers could directly use the downloaded data without having to go through CSV import settings in Excel.

## Current Solution / Workarounds

Either educate the customer to work with CSV files, or send them (e.g. per Email) Excel files that we generate for them by opening the CSV in Excel and saving it as XLSX.

## Alternatives

Some server-side service that could generate XLSX for a given Dataset?

**Comment by mcrascal** [comment]
Doesn't excel open .csv files natively on windows? 

I don't have a PC with the office suite handy, but as far as I recall you could double click a csv file and it would open it in excel.

**Comment by smilingthax** [comment]
AFAIUI, the request came up because of the User Experience, not the general ability to load CSV.

- Technically, you're right: Excel can open CSV files, and should also be associated with the extension in the default configuration, but:
- CSVs can have different Delimiters, Quote characters, Encodings, etc.; getting them wrong results in a more-or-less unusable import.
- e.g. with LibreOffice, for each CSV file it shows an Import Dialog, where you have to make the right choices...
- I'm not too familiar with the current MS Office versions, but according to this post: https://superuser.com/questions/606272/how-to-get-excel-to-interpret-the-comma-as-a-default-delimiter-in-csv-files the expected separator depends on the "system's regional settings". 

As our customers are generally not very tech-savy, XLSX files result in a much better experience.
